### DISCUSSION TITLE
Unable to add gatus pagerduty integration

### DISCUSSION THREAD

**Discussion by glitchcrab** [TwiN/gatus#769] [General]
I've been trying to set up alerting via [pagerduty](https://github.com/TwiN/gatus/blob/master/docs/pagerduty-integration-guide.md#integrating-with-a-pagerduty-service), but I'm unable to complete the setup. From the doc:

> selecting Gatus as the Integration Type in step 4

When searching for 'gatus', there are no results found:

![pagerduty](https://github.com/TwiN/gatus/assets/7832341/19dcfcd8-62f4-45c0-a90f-df68e52bbb42)

Am I missing something? Or is the doc out of date?

**Comment by TwiN** [comment]
Works for me:
![Gatus PagerDuty integraion](https://github.com/TwiN/gatus/assets/15699766/bd93d6b6-3035-4bba-a9f1-5fb118cdea8e)

But I created a ticket with them to double check 🤔

**Reply by glitchcrab** [reply]
that's bizarre - thanks for following up!

**Comment by glitchcrab** [comment]
@TwiN any joy getting an answer from pagerduty?

**Reply by glitchcrab** [reply]
ping @TwiN  :)

**Reply by TwiN** [reply]
Still no luck. Pagerduty hasn't historically been very responsive to me 😅

**Reply by TwiN** [reply]
I do want to point out, however, that it does still work for me and gatus is still in the list of integrations, so I'm thinking perhaps you're doing something wrong 🤔

It'd be nice if we could have a 3rd person weigh in to confirm

**Reply by glitchcrab** [reply]
sure, I did think that it might be a me problem, but i can't really see how - i've been through the docs multiple times and as far as i can tell i'm following them to the letter. the only think i can think of is that my pagerduty account is new (set up specifically for this purpose) and i wonder if that's the reason (although i can't think why it would be).
